locales/en.default.schema.json

```diff
@@ -619,6 +608,9 @@
           "default": "Collections",
           "label": "Heading"
         },
+         "header_layout": {
+        "content": "Layout"
+        },        
```

Comment:
We have many references to `Layout` across the file. There could be an opportunity in consolidating them like we do for `sections.all.padding` etc. 
But it's not something that needs to happen now, it will mess with the translations we're getting back and add some more wait time.

Reply:
Yes good call - it would make sense to unify it